src-tauri/src/lib.rs

```diff
@@ -294,9 +297,28 @@ pub fn launch_northstar(
     }
 
     // Only Windows with Steam or Origin are supported at the moment
-    if host_os == "windows"
-        && (matches!(game_install.install_type, InstallType::STEAM)
-            || matches!(game_install.install_type, InstallType::ORIGIN)
+    if matches!(game_install.install_type, InstallType::STEAM)
+    {
+        // Create run_northstar.txt for NorthstarProton
+        if host_os != "windows"
+        {
+            let mut file = std::fs::OpenOptions::new()
+                .create(true)
+                .write(true)
+                .truncate(true)
+                .open("run_northstar.txt").unwrap();
+
+            let mut content: &[u8] = b"1";
+            std::io::copy(&mut content, &mut file).unwrap();
+        }
+
+        return match open::that("steam://run/1237970/") {
+            Ok(()) => Ok("Launched game".to_string()),
+            Err(_err) => Err(anyhow!("Failed to launch Titanfall 2 via Steam").to_string()),
+        }
```

Comment:
Should work on any System where Steam handles its own scheme (Tested on Windows and Fedora 37).

Another implementation (that would not add another explicit dependency) would be finding Steam (via steamlocate) and executing executing `steam.exe`/`steam.sh` with that.

Reply:
Might need someone to confirm whether Steam Flatpak will also work properly with URI. I'll ask around in the Linux channel on the Northstar Discord.

Personally I was planning to only use URIs too once I'd get around to implementing Linux launch support. So unless it turns out that it decent work for a decent chunk of Linux users, I'm fine with doing it just via URIs ^^

Reply:
So far seems like it works for people in the Linux channel ^^